sources/packages/web/src/components/generic/Banner.vue

```diff
@@ -0,0 +1,30 @@
+<template>
+  <v-alert
+    class="v-alert--variant-contained-text"
+    :type="color"
+    variant="outlined"
+  >
+    <template v-slot:title>
+      <div class="label-bold-normal">{{ header }}</div>
+    </template>
+    <div class="label-value-normal w-75">{{ summary }}</div>
+    <div class="float-right mt-n12">
```

Comment:
If the container arrangement is right, we do not have to use mt-n12. I guess this css class is used to make the action button appear vertically centered. Adding margin to vertically center will have potential side-effects.